src/Presentation/HabitTracker.Presentation/App.xaml.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace HabitTracker.Presentation
+{
+    public partial class App : Application
```

Comment:
`Microsoft.Maui.Controls.Application` conflicts with namespace `Application`.
```suggestion
    public partial class App : Microsoft.Maui.Controls.Application
```